gnome-base/gnome-panel/gnome-panel-3.46.0.ebuild

```diff
@@ -0,0 +1,83 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit gnome2 toolchain-funcs
+
+DESCRIPTION="The GNOME Flashback Panel"
+HOMEPAGE="https://gitlab.gnome.org/GNOME/gnome-panel/"
+
+LICENSE="GPL-2+ FDL-1.1 LGPL-2.1+"
+SLOT="0"
+IUSE="eds elogind systemd"
+REQUIRED_USE="^^ ( elogind systemd )"
+KEYWORDS="~amd64 ~riscv"
+
+# <libgweather-4.2.0 because of libsoup:3 transition
+# <eds-3.46.0 because of build failure https://gitlab.gnome.org/GNOME/gnome-panel/-/issues/41
```

Comment:
Probably can't land this until this is resolved, because we can't depend on both libsoup:2.4 and libsoup:3.0.

Reply:
So I figured out that this only happens, when you build eds-3.46.0 against libsoup-2.4 and then with USE="-weather" (otherwise it wont build because libgweather-4.2.0 build against libsoup-3 is missing)
So how do I forge this into a dependency? 
I could
- directly depend on libsoup-3 and hope eds-3.46.0 was build against libsoup-3 as well. Would probably hit most if not all users with this, but there is the possibility that someone build eds-3.46.0 with USE="-weather" against libsoup-3, installed libsoup-3 later and then tries to emerge gnome-panel.
- depend an eds[weather], so we make sure eds is build with libgweather-4.2.0 and therefore must have been build against libsoup-3 (although it does not really need weather in eds)
- both of the above to make clear gnome-panel needs libsoup-3 (although itself it does not)